cluster/log-dump/log-dump.sh

```diff
@@ -42,7 +42,7 @@ readonly node_ssh_supported_providers="gce gke aws"
 readonly gcloud_supported_providers="gce gke"
 
 readonly master_logfiles="kube-apiserver.log kube-apiserver-audit.log kube-scheduler.log kube-controller-manager.log etcd.log etcd-events.log glbc.log cluster-autoscaler.log kube-addon-manager.log konnectivity-server.log fluentd.log kubelet.cov"
-readonly node_logfiles="kube-proxy.log fluentd.log node-problem-detector.log kubelet.cov"
+readonly node_logfiles="kube-proxy.log fluentd.log node-problem-detector.log kubelet.cov coredns.log"
```

Comment:
What reason do you have to think this is the name of the log?

Reply:
Also: the other logs listed here will exist on *every* node. The in cluster DNS is N pods, not a daemonset

Reply:
yeah. So we use `kubectl logs` to logdump coredns logs ? @BenTheElder

Reply:
This script copies logs from the `/var/log` directory. Adding coredns.log here assumes that there is a `/var/log/coredns.log` present on a node. I believe it's not a correct assumption. To make it work you should change to coredns addon to write logs to `/var/log`.

The other thing is that your are making the change in the deprecated directory. See the README.md file in the log-dump dir. You should make the change in the test-infra repo - https://github.com/kubernetes/test-infra/blob/6505d464406ee67c25fa67ec75de802cb231a753/logexporter/cluster/log-dump.sh

@tosi3k ^

@BenTheElder, it's fine if the file is not present on all nodes. In the scale tests, we already dump "cl2-*" (via setting the `LOG_DUMP_EXTRA_FILES` env) logs that are not present on every node.